### PR TITLE
add some color on ls output

### PR DESCRIPTION
**Pull request by aureliencarle**
A little PR, it's not the most essential but well ...

### CODE REVIEW

File: src/utils/io.py [hunk 1]
```diff
@@ -13,11 +13,19 @@ class DisplayConfig:
 
 class Format:
     @staticmethod
-    def colored(text: str, color: str) -> str:
+    def colored(text: str, color: str, style: str = 'normal') -> str:
```
Comment (GregoireUhlrich) [inline src/utils/io.py]: Maybe as you add a style and not only a color, I would replaced the function name by `styled()`

File: src/utils/io.py [hunk 2]
```diff
@@ -13,11 +13,19 @@ class DisplayConfig:
...
-        fore_color = getattr(Fore, color.upper())
-        fore_reset = Fore.RESET
-        return f'{fore_color}{text}{fore_reset}'
+        color_str = getattr(Fore, color.upper())
+        style_str = getattr(Style, style.upper())
+        reset_str = Style.RESET_ALL
+        return f'{color_str}{style_str}{text}{reset_str}'
+
+    @staticmethod
+    def colored_list(
```
Comment (GregoireUhlrich) [inline src/utils/io.py]: Same as above, would replace by `styled_list()`

**Comment by GregoireUhlrich** [comment]
Not the most essential but well done

**Comment by aureliencarle** [comment]
Do you think we have to constrain colors or let choice to user as config ?

**Comment by aureliencarle** [comment]
Do not forget that in utils directory __init_.py give you the possibility to call class in utils import.
from src.utils.Io import Format => from src.utild import Format

**Comment by GregoireUhlrich** [comment]
Yes the auto imports do this (explicit import without using `__init__.py`)

**Comment by GregoireUhlrich** [comment]
> Do you think we have to constrain colors or let choice to user as config ?

Yes we could. Colors should then be defined separately as configurable variables. Separate them from the code that use them can be done even without configuration as a first step.

Then once colors are decoupled from their use, we can setup a configuration procedure (we'll have to discuss it)

**Pull request merged**